### PR TITLE
cloudflared: update to version 2022.10.3

### PR DESCRIPTION
**Pull request by marmeladema**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/cloudflared/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ license="Apache-2.0"
 homepage="https://developers.cloudflare.com/argo-tunnel/"
 changelog="https://raw.githubusercontent.com/cloudflare/cloudflared/master/CHANGES.md"
 distfiles="https://github.com/cloudflare/cloudflared/archive/${version}.tar.gz"
-checksum=4919eef5814b4ecc41e1359f9546389688194c9e893b4de3c40b0e3cd2919605
+checksum=e49db875da7513d8ae950b20c636225016022866850ff3df2484c0f56cdc4bc4
 
 pre_build() {
 	go_ldflags+=" -X \"main.BuildTime=$(date -u '+%Y-%m-%d-%H:%M UTC')\""
```
[the remark this answers is not in the document]
Comment (marmeladema) [reply srcpkgs/cloudflared/template:18]: According to @ericonr in https://github.com/void-linux/void-packages/pull/26888#discussion_r723785148

> If you use date inside some of the pre_ functions then it uses SOURCE_DATE_EPOCH automatically.

We agreed at the time that it was fine to use `date -u` here. Additionally, I prefer having the correct build date when building locally.

File: srcpkgs/cloudflared/update [hunk 2]
```diff
@@ -0,0 +1 @@
+site=https://github.com/cloudflare/cloudflared/releases
```
Comment (paper42) [inline srcpkgs/cloudflared/update:1]: Is this really needed? update-check detects 2022.10.3 for me without this.

Comment (marmeladema) [reply srcpkgs/cloudflared/update:1]: Definitely not if it works already :D I just thought with an update file, the package would get automatically updated?

Comment (paper42) [reply srcpkgs/cloudflared/update:1]: The update file is used for [checking for new upstream releases](https://github.com/void-linux/void-packages/blob/master/Manual.md#checking-for-new-upstream-releases), every day all packages are checked and the result is available on https://repo-default.voidlinux.org/void-updates/void-updates.txt. Packages do not ever get automatically updated, they have to be bumped and tested manually. Updating them automatically would only lead to low quality packages and a lot of issues.

Comment (marmeladema) [reply srcpkgs/cloudflared/update]: All right, thanks for the detailed explanation :)
Just removed the file.

**Pull request merged**